x-pack/plugins/security_solution/common/detection_engine/utils.ts

```diff
@@ -60,6 +60,9 @@ export const normalizeThresholdField = (
       [thresholdField!];
 };
 
+export const isEqlSequenceQuery = (ruleQuery: string | undefined): boolean =>
+  ruleQuery?.trim().startsWith('sequence') ?? false;
```

Comment:
Correct me if I'm wrong, but it looks like this check is done only on the FE side. Do we have any additional guards/validation on the BE side against creating a sequence EQL rule with suppression via the API? If we don't, what happens if such a rule is created and started?

UPD: Oh I see, in this case the rule will run without suppression and log a warning status 👍 

```ts
result.warningMessages.push(
  'Suppression is not supported for EQL sequence queries. The rule will proceed without suppression.'
);
```

I think it would be worth mentioning in https://github.com/elastic/security-docs/issues/4977

Reply:
Hey @rylnd - how is this warning surfaced? Is it returned in an API response when users create and enable an EQL rule via the [create rule endpoint](https://www.elastic.co/guide/en/security/current/rules-api-create.html#_response_payload)? Does the warning ever appear the Security app UI —  and if so, where? For instance, does it show up on the **Execution results** [tab](https://security-docs_bk_5057.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-monitor.html#rule-execution-logs) within a rule's details page? Or does it appear within the rule creation form when users are setting up a new EQL rule that's using a sequence query? I'm mainly trying to understand whether this warning will be immediately visible to users or if they'll need to seek it out.